Tests/regressionTests.js

```diff
@@ -0,0 +1,301 @@
+        .then(() => itemPage.getUserName())
+        .then((data) => expect(data).toBe("Rafa Nadal"))
+        .then(() => itemPage.getBidPrice())
+        .then((data) => expect(data).toBe("$400.25.00"))
+        .then(() => itemPage.getBidDate())
+        .then((data) => expect(data).toBe(`${d.getDate()} ${months[d.getMonth()]} ${d.getFullYear()}`))
+    });
+    it("002: User is able to make a bid with round values", function(){
+        var d = new Date();
+        var months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+        homepage.clickOnFashionCategory()
+        .then(() => browser.wait(EC.presenceOf($(".landing-product")), 5000))
+        .then(() => fashionCategory.clickOnBlueJacket())
+        .then(() => itemPage.enterBid(5000))
+        .then(() => itemPage.clickOnPlaceBidButton())
+        .then(() => browser.wait(EC.presenceOf($(".bider-name-img")), 5000))
+        .then(() => itemPage.getUserName())
+        .then((data) => expect(data).toBe("Rafa Nadal"))
+        .then(() => itemPage.getBidPrice())
+        .then((data) => expect(data).toBe("$5000.00"))
+        .then(() => itemPage.getBidDate())
+        .then((data) => expect(data).toBe(`${d.getDate()} ${months[d.getMonth()]} ${d.getFullYear()}`))
+    })
+    it("003: User isn't able to make bid which is lower than starting price", function(){
+        homepage.clickOnFashionCategory()
+        .then(() => browser.wait(EC.presenceOf($(".landing-product")), 5000))
+        .then(() => fashionCategory.clickOnWhiteJacket())
+        .then(() => itemPage.enterBid(50))
+        .then(() => itemPage.clickOnPlaceBidButton())
+        .then(() => browser.wait(EC.presenceOf($(".status-info")), 5000))
+        .then(() => itemPage.getBidConfirmationText())
+        .then((data) => expect(data).toBe("There are higher bids than yours. You could give a second try!"))
+    })
+})
+describe("005: Abouts Us", function (){
+    beforeEach(() => {
+        homepage.open("https://auct-app2-frontend.herokuapp.com/")
+        })
+    it ("001: User is able to open 'About Us' section", function (){
+        homepage.clickOnAboutUs()
+        .then(() => browser.wait(EC.presenceOf($(".helper")), 5000))
+        .then(() => aboutUs.getAboutParagraph())
+        .then((data) => expect(data).toBe("About Us") )
+    })
+});
+describe("006: Terms and Conditions", function (){
+    beforeEach(() => {
+        homepage.open("https://auct-app2-frontend.herokuapp.com/")
+        })
+    it ("001: User is able to open 'Terms and Conditions' section", function (){
+        homepage.clickOnTermsAndConditions()
+        .then(() => browser.wait(EC.presenceOf($(".helper")), 5000))
+        .then(() => termsandconditions.getIntroductionParagraph())
+        .then((data) => expect(data).toBe("Introduction"))
+    })
+})
+describe("007: Privacy and Policy", function (){
```

Comment:
For every test suite you should have different .js class created.
One test.js should have one describe (test suites) and more related its (test cases).

Reply:
I created different classes for test suites

Reply:
Not fixed yet

Reply:
Sorry, now every test suite has it's own class with name (ex testSuiteTests.js) in folder Tests